### PR TITLE
[Adjustment][Maxell] optimized login/signup change buttons

### PR DESCRIPTION
**Pull request by maxellmilay**
- adjusted such that if login is asked, the sign up button will be available, vice-versa

### CODE REVIEW

**Comment by maxellmilay** [comment]
image
image

File: client/src/pages/UserLogin/components/LogInContainer.js [hunk 1]
```diff
@@ -74,36 +74,39 @@ const LogInContainer = (props) => {
...
-              variant="contained"
-              type="button"
-              size="small"
-              sx={buttonTextSize}
-              onClick={handleChange}
-            >
-              Sign Up
-            </Button>
-          </Grid>
+          {!isLoggingIn ? (
```
Comment (ezerssss) [inline client/src/pages/UserLogin/components/LogInContainer.js:77]: avoid negative conditionals if possible, i switch nalang ni sila

File: client/src/pages/UserLogin/components/LogInContainer.js [hunk 2]
```diff
@@ -74,36 +74,39 @@ const LogInContainer = (props) => {
...
+                color="secondary"
+                variant="contained"
+                size="small"
+                type="button"
+                sx={buttonTextSize}
+                onClick={handleChange}
+              >
+                Log In
+              </Button>
+            </Grid>
```
Comment (ezerssss) [inline client/src/pages/UserLogin/components/LogInContainer.js:92]: don't repeat yourself principle bai, gibalik ra nimo basically ang component pero different texts lang, how about just create these texts as a variable nya didto lang i perform ang logic
ex:
`footerText = isLoggingIn? "Don't have an account?" : "Login to access site"` 

nya same lang for the button maybe `footerButton` or kanang better name if maka think ka

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: client/src/pages/UserLogin/components/LogInContainer.js (hunk 1 above)
Comment (maxellmilay) [reply client/src/pages/UserLogin/components/LogInContainer.js]: resolved

File: client/src/pages/UserLogin/components/LogInContainer.js (hunk 2 above)
Comment (maxellmilay) [reply client/src/pages/UserLogin/components/LogInContainer.js]: resolved

**Review by ezerssss — APPROVED** [review]

**Pull request merged**